edi_route/i18n/sv_SE.po

```diff
@@ -0,0 +1,925 @@
+msgid "Followers"
+msgstr "Följare"
+
+#. module: edi_route
+#: model:ir.model.fields,field_description:edi_route.field_edi_envelope__message_channel_ids
+#: model:ir.model.fields,field_description:edi_route.field_edi_message__message_channel_ids
+#: model:ir.model.fields,field_description:edi_route.field_edi_route__message_channel_ids
+msgid "Followers (Channels)"
+msgstr "Följare (Kanaler)"
+
+#. module: edi_route
+#: model:ir.model.fields,field_description:edi_route.field_edi_envelope__message_partner_ids
+#: model:ir.model.fields,field_description:edi_route.field_edi_message__message_partner_ids
+#: model:ir.model.fields,field_description:edi_route.field_edi_route__message_partner_ids
+msgid "Followers (Partners)"
+msgstr "Följare (kontakter)"
+
+#. module: edi_route
+#: model:ir.model.fields,field_description:edi_route.field_edi_message__forwarder_id
+#: model_terms:ir.ui.view,arch_db:edi_route.view_edi_message_search
+msgid "Forwarder"
+msgstr ""
+
+#. module: edi_route
+#: model:ir.model.fields,help:edi_route.field_edi_message__forwarder_id
+msgid "Forwarder - the party planning the transport on behalf of the consignor or consignee."
+msgstr ""
+
+#. module: edi_route
+#: model:ir.model.fields,field_description:edi_route.field_edi_route__frequency_quant
+msgid "Frequency"
+msgstr ""
+
+#. module: edi_route
+#: model:ir.model.fields,field_description:edi_route.field_edi_route__frequency_uom
+msgid "Frequency Uom"
+msgstr ""
+
+#. module: edi_route
+#: model_terms:ir.ui.view,arch_db:edi_route.view_edi_envelope_search
+#: model_terms:ir.ui.view,arch_db:edi_route.view_edi_message_search
+#: model_terms:ir.ui.view,arch_db:edi_route.view_edi_route_search
+msgid "Group By"
+msgstr "Gruppera efter"
+
+#. module: edi_route
+#: model:ir.module.category,description:edi_route.module_category_edi_management
+msgid "Helps you handle EDI messages, envelopes and routes.."
+msgstr ""
+
+#. module: edi_route
+#: model:ir.model.fields,field_description:edi_route.field_edi_application_line__id
+#: model:ir.model.fields,field_description:edi_route.field_edi_envelope__id
+#: model:ir.model.fields,field_description:edi_route.field_edi_message__id
+#: model:ir.model.fields,field_description:edi_route.field_edi_message_type__id
+#: model:ir.model.fields,field_description:edi_route.field_edi_route__id
+#: model:ir.model.fields,field_description:edi_route.field_edi_route_caller__id
+#: model:ir.model.fields,field_description:edi_route.field_edi_route_line__id
+msgid "ID"
+msgstr ""
+
+#. module: edi_route
+#: selection:edi.route,protocol:0
+msgid "IPF"
+msgstr ""
+
+#. module: edi_route
+#: model:ir.model.fields,help:edi_route.field_edi_envelope__message_unread
+#: model:ir.model.fields,help:edi_route.field_edi_message__message_unread
+#: model:ir.model.fields,help:edi_route.field_edi_route__message_unread
+msgid "If checked new messages require your attention."
+msgstr "Om ikryssad nya meddelanden som kräver din uppmärksamhet"
```

Comment:
This translation seems to be halting a bit, see translation below for basically the same message